### DISCUSSION TITLE
What is refractor lang package

### DISCUSSION THREAD

**Discussion by theskillwithin** [vercel/next.js#40485] [Help] [answered]
Hi, I did a bundle analyze scan on my Next.js app.   One of the biggest chunks is from node_modules the refractor > lang module.

It seems Next.js is adding this,  is there a way to remove this?  or is this a necessary thing?

<img width="1410" alt="image" src="https://user-images.githubusercontent.com/8095506/189706636-4f5eb4cf-8a9d-4a31-96c6-c53f8f0db5b0.png">

**Comment by theskillwithin** [comment, marked as answer]
found the problem https://gist.github.com/theskillwithin/cbbbd3e01d5a7d9174e36e0026d8f67d

**Reply by Mageshwaran2314** [reply]
facing same issue, this one loads syntax for all the languages 
`import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";`
any solution on reducing the bundle size ?
